README.md

```diff
@@ -15,8 +15,18 @@ This plugin intends to help you in tracking down problems when you are using css
 where container is the css class that you want to mark as used.
 Add all such classes in the array.
 
+>If you use the `camelCase` option of `css-loader`, you must also enabled it for this plugin
+```js
+/* eslint css-modules/no-unused-class: [2, { camelCase: true }] */
+```
+
 * `css-modules/no-undef-class`: You must not use a non existing class.
 
+>If you use the `camelCase` option of `css-loader`, you must also enabled it for this plugin
+```js
+/* eslint css-modules/no-unused-class: [2, { camelCase: true }] */
```

Comment:
Yes I copied from above and forgot to change the rule name.

> Even better, i think there should be example of how to set this in an .eslintrc file.
Because users would want this setting in their complete project.

So perhaps, we could remove the current documentation about the options and put just one code block with both rules and their options?